request_a_govuk_domain/request/models/organisation.py

```diff
@@ -25,9 +15,16 @@ class RegistrantTypeChoices(models.TextChoices):
     representative = "Representing public sector bodies"
 
 
-class Registrant(Organisation):
+class Registrant(models.Model):
+    name = models.CharField()
     type = models.CharField(choices=RegistrantTypeChoices.choices, max_length=100)
 
+    def __str__(self):
+        return self.name
+
 
-class Registrar(Organisation):
-    email_address = models.EmailField(max_length=320)
+class Registrar(models.Model):
+    name = models.CharField()
```

Comment:
I think we can still have unique constraint for Registrar name, because there won't be two records with the same name. Now it doesn't inherit from Organisation, so we can independently maintain unique constraint here, without affecting Registrant.